packages/pigeon/example/app/android/app/src/main/kotlin/dev/flutter/pigeon_example_app/MainActivity.kt

```diff
@@ -37,22 +36,21 @@ private class PigeonApiImplementation : ExampleHostApi {
 // #enddocregion kotlin-class
 
 // #docregion kotlin-class-flutter
-private class PigeonFlutterApi {
-
+private class PigeonFlutterApi(binding: FlutterPlugin.FlutterPluginBinding) {
```

Comment:
This and the constructor->init change were an Android Studio-suggested refactor; I figured I might as well update it to follow recommendations while I was there, but I can back this part out if we don't want it.